docs/03-tutorials/00-api-exposure/apix-04-expose-and-secure-workload-jwt-istio-ext-auth.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Expose and secure a workload with Istio external auth
+---
+
+This tutorial shows how to expose workload using VirtualService and secure it using Istio authorization policy together with Istio RequestAuthentication based on JWT token
```

Comment:
please mention that this tutorial can be follow up after [Use a custom domain to expose a workload](https://kyma-project.io/docs/kyma/main/03-tutorials/00-api-exposure/apix-01-own-domain/)